### PR TITLE
Add missing endpoint for Azure AZ

### PR DESCRIPTION
**Pull request by kgroschoff**
**What this PR does / why we need it**:
Add missing endpoint for Azure AZ that is needed for the add node dialog after cluster creation

**Special notes for your reviewer**:
`SKUName` = Size Name :) 

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: api/pkg/handler/v1/provider/azure.go [hunk 1]
```diff
@@ -354,3 +395,35 @@ func DecodeAzureSizesReq(c context.Context, r *http.Request) (interface{}, error
...
+	req.TenantID = r.Header.Get("TenantID")
+	req.ClientID = r.Header.Get("ClientID")
+	req.ClientSecret = r.Header.Get("ClientSecret")
+	req.Location = r.Header.Get("Location")
+	req.SKUName = r.Header.Get("SKUName")
+	req.Credential = r.Header.Get("Credential")
+	return req, nil
+}
+
+type AzureAvailabilityZonesNoCredentialsReq struct {
```
Comment (zreigz) [inline api/pkg/handler/v1/provider/azure.go:415]: Please add those comments
```
// AzureAvailabilityZonesNoCredentialsReq represent a request for Azure Availability Zones
// note that the request doesn't have credentials for authN
// swagger:parameters listAzureAvailabilityZonesNoCredentials
```

**Comment by kgroschoff** [comment]
Thanks @zreigz , test is working 👍 PTAL :)

**Pull request merged**